### ISSUE TITLE
bug: `Model failed to get model status with status code 409` but model runs successfully

### ISSUE THREAD

**Issue by gabrielle-ong** [janhq/cortex.cpp#1504]
### Cortex version

v172

### Describe the Bug

`cortex run` returns model failed 409 status code. 
But model starts successfully, can successfully chat (CLI, API postman).
This is confusing to users as they may think there is an error
Seems like Louis also got a 409 error in #1475

```
> cortex run model
Starting server ...
Host: 127.0.0.1 Port: 39281
Server started
API Documentation available at: http://127.0.0.1:39281
Error: Model failed to get model status with status code: 409
tinyllama:1b-gguf model started successfully. Use `cortex-nightly chat tinyllama:1b-gguf` for interactive chat shell
```

### Screenshots / Logs

<img width="1440" alt="image" src="https://github.com/user-attachments/assets/c29e714b-7a36-4b45-b1cc-17503f4582c6">

### What is your OS?

- [X] MacOS
- [ ] Windows
- [ ] Linux

### What engine are you running?

- [X] cortex.llamacpp (default)
- [ ] cortex.tensorrt-llm (Nvidia GPUs)
- [ ] cortex.onnx (NPUs, DirectML)

**Comment by gabrielle-ong** [comment]
![Image](https://github.com/user-attachments/assets/34da9600-206b-40e1-aaa3-a0e73f37cc1b)
Also getting with HF model

**Comment by vansangpfiev** [comment]
@gabrielle-ong This should be fixed by `v1.0.0-178`, can you please try again?

**Comment by gabrielle-ong** [comment]
Confirmed resolved, thanks @vansangpfiev. Can you tag which PR solved this?
